sections/04_study.md

```diff
@@ -34,34 +34,33 @@ Gene expression technologies characterize the abundance of many thousands of RNA
 transcripts within a given organism, tissue, or cell. This characterization can
 represent the underlying state of the given system and can be used to study
 heterogeneity across samples as well as how the system reacts to perturbation.
-While gene expression measurements have been traditionally made by quantitative
+While gene expression measurements were traditionally made by quantitative
 polymerase chain reaction (qPCR), low-throughput fluorescence-based methods, and
 microarray technologies, the field has shifted in recent years to primarily
 performing RNA sequencing (RNA-seq) to catalog whole transcriptomes. As RNA-seq
-continues to fall in price and rise in throughput, applying deep learning to
-study gene expression data is likely to make training deep models more feasible.
-With increased modeling ability, deep learning approaches are likely to grow in
-popularity and lead to novel biological insights.
+continues to fall in price and rise in throughput,
```

Comment:
Maybe my ignorance, but RNASeq getting faster and cheaper doesn't make deep models more "feasible". More useful, maybe? More interesting? Unless there's the idea of gathering huge amounts of data to make models across the whole transcriptome.